shared/constants/uns.js

```diff
@@ -0,0 +1,16 @@
+export const ALLOWED_UNSTOPPABLE_TLDS = [
```

Comment:
We can pull the list from `GET https://resolve.unstoppabledomains.com/supported_tlds` to make sure the TLDs are up-to-date.

Reply:
Currently working on getting this implemented :D